Misc/NEWS

```diff
@@ -10,6 +10,9 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-29960: Preserve generator state when _random.Random.setstate()
```

Comment:
This entry should be added in the "Library" section rather than "Core and Builtins".